data/controller_data.py

```diff
@@ -84,6 +84,11 @@
      'image_url': 'http://zero88.com/images/product-pics-scd-server.jpg',
      'tags': ['console']},
   ],
+  0x2239: [
+    {'name': 'LightFactory',
+     'link': 'https://www.lightfactory.net/',
+     'image_url': 'https://www.lightfactory.net/images/lightfactory_square1_small.png'},
+  ],
```

Comment:
You probably want to add a tags line with console in it @msearancke so it shows up in this list: http://rdm.openlighting.org/controller/browse?tag=console

Reply:
Thanks, done.